crates/cxx-qt-gen/src/parser/cxxqtdata.rs

```diff
@@ -66,23 +66,36 @@ pub struct ParsedCxxQtData {
 
 impl ParsedCxxQtData {
     /// Find the QObjects within the module and add into the qobjects BTreeMap
-    pub fn find_qobject_structs(&mut self, items: &[Item]) -> Result<()> {
+    pub fn find_qobject_types(&mut self, items: &[Item]) -> Result<()> {
         for item in items {
-            if let Item::Struct(qobject_struct) = item {
-                if let Some(index) =
-                    attribute_find_path(&qobject_struct.attrs, &["cxx_qt", "qobject"])
+            if let Item::ForeignMod(foreign_mod) = item {
+                if foreign_mod.abi.name.as_ref().map(|lit_str| lit_str.value())
+                    == Some("RustQt".to_string())
                 {
-                    // Load the QObject
-                    let mut qobject = ParsedQObject::from_struct(qobject_struct, index)?;
-
-                    // Inject the bridge namespace if the qobject one is empty
-                    if qobject.namespace.is_empty() && !self.namespace.is_empty() {
-                        qobject.namespace = self.namespace.clone();
+                    for foreign_item in &foreign_mod.items {
+                        if let ForeignItem::Verbatim(tokens) = foreign_item {
+                            let foreign_alias: ForeignTypeIdentAlias = syn::parse2(tokens.clone())?;
+
+                            // TODO: in the future qobject macro will be removed and all types in RustQt will be QObjects
```

Comment:
We'll need to move stuff currently specified with the `#[qobject]` macro like `base_class` to another syntax, perhaps make them their own attributes like `#[base_class(QAbstractItemModel)]` or something